river/command/declare_mode.zig

```diff
@@ -39,9 +39,9 @@ pub fn declareMode(
 
     if (config.mode_to_id.get(new_mode_name) != null) return;
 
-    try config.modes.ensureUnusedCapacity(1);
+    try config.modes.ensureUnusedCapacity(util.gpa, 1);
```

Comment:
Here `util.gpa`, as in the lines below, is used when in `command/map.zig` (see below) the `allocator` args is used, I feel like this is kinda bad consistency but not sure which one to chose.